Cargo.toml

```diff
@@ -23,12 +23,18 @@ tokio-io = "0.1"
 tokio-codec = "0.1"
 tokio-sync = "0.1"
 
+[features]
+default = [ "geospatial" ]
```

Comment:
Ah hm, turns out we don't necessarily need to disable the code, but only the test.
What do you think?

Reply:
I saw the suggestion in https://github.com/mitsuhiko/redis-rs/pull/130#issuecomment-478644177, but I think that it can be confusing for users of the crate. In other crates that I have seen, *features* are something intended for users of the library, and not an internal detail of the implementation (like configuring the CI).

A `geospatial` feature is easier to recognise by users, and it is still possible to control the CI.

Reply:
Makes sense.
I'm fine with that, as long as the feature is enabled by default.